### DISCUSSION TITLE
how  i can connect kafka ui with multiple kafka cluster

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1630] [Q&A] [answered]
Right now i'm using below command to connect to kafka cluster , if i want to connect to multiple kafka cluster with this . how can i do . 

 docker run -p 9095:8080 \
    -e KAFKA_CLUSTERS_0_NAME=isds-kafkaservice  \
    -e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=isds-kafkaservice-dev-west2-lkf01:9094 \
    -e KAFKA_CLUSTERS_0_ZOOKEEPER=isds-kafkaservice-dev-west2-lzk01:2185 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL=SASL_SSL \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM=SCRAM-SHA-512 \
    -e KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG="org.apache.kafka.common.security.scram.ScramLoginModule required username="kafka_admin" password="xxxx";" \
    -d  provectuslabs/kafka-ui:latest

**Comment by Haarolean** [comment, marked as answer]
[kafka-ui.yaml](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui.yaml) contains a complex example with multiple clusters (2, actually). You have to do the same, but replace "0" in the env. variable names with "1".
`KAFKA_CLUSTERS_0_NAME`;
`KAFKA_CLUSTERS_1_NAME`, etc.
